templates/download/cloud/index.html

```diff
@@ -9,7 +9,7 @@
   <div class="row u-equal-height">
     <div class="col-8">
       <h1>Optimised Ubuntu on&nbsp;public&nbsp;clouds</h1>
```

Comment:
maybe "Optimised" has been removed? 
```suggestion
      <h1>Ubuntu on&nbsp;public&nbsp;clouds</h1>
```

Reply:
Checked with Lech, "Optimised" should be there, so I've added it to the copy doc.